docs/data/data-grid/components/export/export.md

```diff
@@ -0,0 +1,37 @@
+# Data Grid - Export component
+
+<p class="description">Components to trigger exports of the Data Grid.</p>
+
+## Anatomy
+
+The Export component is comprised of the following parts:
+
+```tsx
+<Grid.Export.Trigger />
+```
+
+### Trigger
+
+A button that triggers an export.
+
+The `exportType` prop can be set to `print`, `csv` or `excel`[<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan').
+
+## Examples
+
+Below are some ways the Export component can be used.
+
+### Toolbar export buttons
+
+Display export options as buttons on the toolbar.
+
+{{"demo": "GridExportTrigger.js", "bg": "inline"}}
+
+### Toolbar export menu
+
+Display export options within a menu on the toolbar.
+
+{{"demo": "GridExportMenu.js", "bg": "inline"}}
+
+## API
+
+- TBC
```

Comment:
Having issues generating the API page for this currently, as there is a `GridExportTrigger` exported from both the community and premium package. I think having a single component which can take different `exportType` values depending on the license is the best DX here, so keen not to lose that.

Reply:
Side note: I noticed that `GridExportTrigger` in the premium package reimplements the one from the community package + adds Excel export. So the component from the premium package won't inherit the changes made to the community one, and it's easy to miss.

Did you consider having separate components for each export type? In this case, we'll export an additional trigger component for Excel export. What do you think?

Reply:
> So the component from the premium package won't inherit the changes made to the community one, and it's easy to miss.

Fair point - I've split it out into three separate components to avoid this.